benchmark/flags/flags.go

```diff
@@ -138,3 +139,41 @@ func (isv *intSliceValue) String() string {
 	}
 	return b.String()
 }
+
+type stringSliceValue []string
+
+// StringSlice returns a flag representing a slice of strings. Commas will be
+// interpreted as separators unless they are escaped by a backslash.
```

Comment:
This says "backslash" but the comment below says "double-quotes" -- is this right?

Reply:
Done. Removed the comment altogether; the double quotes one is the correct one.